app/api/data.ts

```diff
@@ -0,0 +1,26 @@
+export const data = [
```

Comment:
move api folder from app folder

Reply:
I can't move the api folder. If I do that, I can't access the api folder